modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_content_marquee.yml

```diff
@@ -20,34 +20,40 @@ process:
     process:
       delta: delta
       value: value
-  
+
   field_az_media:
     plugin: sub_process
     source: field_uaqs_photo
     process:
       delta: delta
-      target_id: 
-      - 
+      target_id:
+      -
         plugin: migration_lookup
         source: fid
-        migration: 
+        migration:
           - az_media
 
-  field_az_text_area/value: 
+  field_az_text_area/value:
     plugin: paragraphs_fw_media_row_field_merge
     body: field_uaqs_summary
     link: field_uaqs_link
 
-  field_az_text_area/format:  
+  field_az_text_area/format:
     plugin: default_value
     default_value: az_standard
 
+  field_uaqs_setting_bg_attach_value: field_uaqs_setting_bg_attach/0/value
```

Comment:
`field_uaqs_setting_bg_attach/0/value`  could probably just be on line 54 instead of making this pseudo destination.